terraform/pipeline_ecs.tf

```diff
@@ -149,18 +149,18 @@ resource "aws_security_group" "pipeline_ecs_service_sg" {
   }
 }
 
-resource "aws_ecs_service" "pipeline_service" {
-  name            = "c14-trendgineers-pipeline-service"
-  cluster         = data.aws_ecs_cluster.cluster.id
-  task_definition = aws_ecs_task_definition.c14_trendgineers_pipeline.arn
-  desired_count   = 1
-  launch_type     = "FARGATE"
-
-  network_configuration {
-    subnets         = ["subnet-0497831b67192adc2",
-                "subnet-0acda1bd2efbf3922",
-                "subnet-0465f224c7432a02e"] 
-    security_groups = [aws_security_group.pipeline_ecs_service_sg.id]
-    assign_public_ip = true
-  }
-}
\ No newline at end of file
+# resource "aws_ecs_service" "pipeline_service" {
+#   name            = "c14-trendgineers-pipeline-service"
+#   cluster         = data.aws_ecs_cluster.cluster.id
+#   task_definition = aws_ecs_task_definition.c14_trendgineers_pipeline.arn
+#   desired_count   = 1
+#   launch_type     = "FARGATE"
+
+#   network_configuration {
+#     subnets         = ["subnet-0497831b67192adc2",
+#                 "subnet-0acda1bd2efbf3922",
+#                 "subnet-0465f224c7432a02e"] 
+#     security_groups = [aws_security_group.pipeline_ecs_service_sg.id]
+#     assign_public_ip = true
+#   }
+# }
```

Comment:
do we need this commented out code?

Reply:
We will need this code, however we don't want to create these services just yet, keeping them commented out to allow us to run terraform apply without worry